lib/slack-ruby-bot-server/api/endpoints/teams_endpoint.rb

```diff
@@ -60,7 +61,9 @@ class TeamsEndpoint < Grape::API
               )
             end
 
-            Service.instance.create!(team)
+            state = params.key?(:state) ? { state: params[:state] } : {}
```

Comment:
Rather that `state` this is `options`, let's just rename.

But just use `slice`, so 

```ruby
options = params.slice(:state)
Service.instance.create!(team, options)
```